src/components/Card.svelte

```diff
@@ -15,23 +24,28 @@
 
 		return artistsOneline;
 	}
+	//when collapsed, pause audio and set expand indicator to false
 	function toggleExpand():void{
+		if(expand){
+			audio.pause();
+		}
 		expand = !expand
 	}
 </script>
 
-<div class="card" on:keyup={toggleExpand}>
-	<div>
+<div class="card" on:click={toggleExpand} on:keyup={toggleExpand} aria-expanded={expand}>
```

Comment:
Why is it when it comes to keyboard navigation, it's onKeyUp, but when it is clicks, it's on click (I think it's on click down). Is there a specific reason? Why not onMouseUp (check accessibility issues - is it usable on touchscreen devices?)